megamek/src/megamek/common/util/StringUtil.java

```diff
@@ -1,33 +1,26 @@
 /*
  * MegaMek - Copyright (C) 2003 Ben Mazur (bmazur@sev.org)
- * 
- *  This program is free software; you can redistribute it and/or modify it 
- *  under the terms of the GNU General Public License as published by the Free 
- *  Software Foundation; either version 2 of the License, or (at your option) 
- *  any later version.
- * 
- *  This program is distributed in the hope that it will be useful, but 
- *  WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY 
- *  or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License 
- *  for more details.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
  */
 package megamek.common.util;
 
+import megamek.codeUtilities.StringUtility;
+import megamek.common.preference.PreferenceManager;
+
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
-import java.util.Comparator;
 import java.util.Vector;
 
-import megamek.common.preference.PreferenceManager;
-
 public class StringUtil {
```

Comment:
Surprise, I had expected for this to be deleted and integrated into StringUtility. Not good?

Reply:
I took the most useful suite-wide methods from the two utilities and moved them into the new utilities setup. I'm going to investigate what's left here and MekHQ's Utilities to see if their functionality can be moved or removed, but felt that would be best left for its own PR.